sip/sip-005-blocks-and-transactions.md

```diff
@@ -239,7 +240,10 @@ The Stacks blockchain supports the following two types of comparators:
   fungible asset balance increased or decreased as a result of the transaction's
   execution.  The contract can assert that the quantity of tokens increased,
   decreased, or stayed the same.  In addition, the contract can assert that the
-  change was greater than, equal to, or less than a given amount.
+  change was greater than, equal to, or less than a given amount.  If any fungible
+  asset post-conditions are present, then such conditions are treated as a whitelist,
+  meaning that any fungible tokens not present in the post-conditions are treated as
+  "stays the same" constraints.
```

Comment:
I don't think we should mandate that it's a whitelist (but I can be convinced otherwise), and I don't think the whitelisting should be limited to fungible tokens.  In my implementation, the transaction can indicate whether or not the postconditions are a whitelist of changes, and can set postconditions on STX, fungible tokens, and non-fungible tokens.

Reply:
Sure -- I think making this an option is fine. I reverted this text.